tech/languages/csharp/csharp-ide.md

```diff
@@ -26,7 +26,7 @@ JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core
 
 _.NET Core and Mono_
 
-You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE. Relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
+You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE, however **it will not work with our dotnet packages** due to an [upstream issue](https://github.com/dotnet/source-build/issues/125). Relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
```

Comment:
**it will not work with our dotnet packages**

V tom pripade bych to vyradil, kdyz to vubec nefunguje na Fedore, protoze tu maji byt jen funkcni veci ...

Reply:
Ono to funguje, ale len s microsoft-built packages - https://www.microsoft.com/net/core#linuxfedora
Dunno.. Still works for mono, even if not for dotnet (with our packages)

Reply:
If it 'works, but ...'- then please do (you have to!) mention it...

The web pages are ment for developers, looking for some quickstart or reference, etc., and therefore it should help user avoid, or resolve, any struggles with installing configuring stuff that you're refering to. It's not a documentation, wiki or alike. It's supposed to contain only relevant information to latest Fedora stable release.

In case of guide, or command, it has to lead to a working setup, even with some workarounds or limitations included(but you have to mention them!).

While the text says `You can use the [aCute plugin]`, you're now actually telling me the user cannot use it, unless he/she installs it from other source.

Please create PR fixing this, or I'll have to delete it altogether.

Reply:
Is this better?

Reply:
Yes, looks quite fine- please see other thread.